app/javascript/components/MainApp.js

```diff
@@ -134,12 +134,8 @@ reloadPage = (id) => {
        { /* <Route   path="/profile" exact render={( ...props) => <Profile edit_user_route={edit_user_route}/> } /> */}
         <Route path="/aboutus" exact component={AboutUs} /> 
        {/* <Route path="/newchannel" exact component={NewChannel} /> */}
-       {logged_in &&
-        <div>
-        {channels &&
+       {(logged_in && channels) &&
```

Comment:
Why are we making this change?

It shouldn't be a problem to have a route existing that never gets used if there are no channels.

Reply:
the div element breaks the switch and wont render the routes under them.
and gives an error like this React does not recognize the `computedMatch` prop on a DOM element.